src/Arcus.Templates.AzureFunctions.Databricks.JobMetrics/.template.config/ide.host.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "learnMoreLink": "https://templates.arcus-azure.net/features/web-api-template",
+  "icon": "icon.jpg",
+  "symbolInfo": [
```

Comment:
Let's add these as well:
- `description`
- `supportsDocker`
- `isApi`

Reply:
While `suportsDocker` is useful, we already added this by default. This will result in a checkbox that will add the Docker support (which we already added).

Reply:
So this is an extension of our regular template configuration then or how should I see it?

Reply:
Added `name`, `description` and `isApi` (on web API and Azure Functions).

Reply:
> So this is an extension of our regular template configuration then or how should I see it?

Yes, this will be used when the project template is used via VS but it's only a 'bridge' between the two. You describe what you want Visual Studio to show and what to hide.